src/arrays/referencing_array_and_its_elements/hard/q9/Lottery.java

```diff
@@ -0,0 +1,28 @@
+package arrays.referencing_array_and_its_elements.hard.q9;
+
+/*
+Write a Java program that randomly generates an array of integers with 7 elements. Each element should have 5 digits
```

Comment:
Because of our discussion in the discord on forcing users to make arrays instead of getting around it just using a few variables we're going to have to kick up the number from 7 to somewhere between 100-500